ipapython/ipa_log_manager.py

```diff
@@ -164,6 +238,7 @@ def standard_logging_setup(filename=None, verbose=False, debug=False,
         finally:
             os.umask(umask)
         file_handler.setLevel(logging.DEBUG)
+        file_handler.addFilter(InfoCommandFilter())
```

Comment:
Why do you add the filter to the file logger instead of the console logger? Isn't the filter suppose to filter which events end up in stdout?

Reply:
Adding the filter to the installation log fil e handler makes sure that the print output is not in the log file (like it has been up to now) and is by default in the normal installer (log) output. This makes sure that an existing application using FreeIPA parts does not need to enable this explicitly.

Reply:
I think we have some kind of misunderstanding. Having the output in the log files is one of the benefits for me. It makes it easier to correlate debug logs with installer console output. 

I proposed the ``cmd_output`` flag to mark log entries that **also** land on stdout (console logger).

Reply:
This is behavior change that needs to be verified also in all the tests.